apps/condo/domains/user/schema/ChangePhoneNumberResidentUserService.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Generated by `createservice user.ChangePhoneNumberResidentUserService`
+ */
+
+const { GQLCustomSchema } = require('@core/keystone/schema')
+const access = require('@condo/domains/user/access/ChangePhoneNumberResidentUserService')
+const {
+    CONFIRM_PHONE_ACTION_EXPIRED,
+} = require('@condo/domains/user/constants/errors')
+const { ConfirmPhoneAction, User } = require('@condo/domains/user/utils/serverSchema')
+
+
+const ChangePhoneNumberResidentUserService = new GQLCustomSchema('ChangePhoneNumberResidentUserService', {
+    types: [
+        {
+            access: true,
+            type: 'input ChangePhoneNumberResidentUserInput { dv: Int!, sender: JSON!, token: String! }',
+        },
+        {
+            access: true,
+            type: 'type ChangePhoneNumberResidentUserOutput { status: String! }',
+        },
+    ],
+
+    mutations: [
+        {
+            access: access.canChangePhoneNumberResidentUser,
+            schema: 'changePhoneNumberResidentUser(data: ChangePhoneNumberResidentUserInput!): ChangePhoneNumberResidentUserOutput',
+            resolver: async (parent, args, context, info, extra = {}) => {
+                const { data: { token } } = args
+                const userId = context.authedItem.id
+                const [action] = await ConfirmPhoneAction.getAll(context,
+                    {
+                        token,
+                        completedAt: null,
+                        isPhoneVerified: true,
+                    }
+                )
+                if (!action) {
+                    throw new Error(`${CONFIRM_PHONE_ACTION_EXPIRED}] Unable to find confirm phone action`)
+                }
+                const { phone: newPhone } = action
+                await User.update(context, userId, { phone: newPhone })
```

Comment:
add phone normalization/validation

Reply:
This phone is from startConfirmPhoneAction mutation it has normalize there - here we only fetch it from database